core/arch/arm/kernel/boot.c

```diff
@@ -82,7 +82,7 @@ struct dt_descriptor {
 #endif
 };
 
-static struct dt_descriptor external_dt __nex_bss;
+static struct dt_descriptor external_dt __nex_bss __maybe_unused;
```

Comment:
`__maybe_unused` not needed since access to the resource is conditioned with `if (...IS_ENABLED(...)...)` instead of `#if`/`#endif`.

Reply:
Indeed